src/types/openshift.ts

```diff
@@ -26,19 +26,3 @@ export type Release = {
   url?: string;
   channels?: string[];
 };
```

Comment:
We can remove `UpdateHistory` and `Release` type definitions.

Reply:
Updated the PR